tests/generative/variable_strategies.py

```diff
@@ -408,11 +408,16 @@ def filtered_table(draw):
 
     @st.composite
     def sorted_frame(draw):
-        # select a table which may already have been filtered
-        source = draw(many_rows_per_patient_frame())
-        # Now apply 1-3 sorts
-        for _ in range(draw(st.integers(min_value=1, max_value=3))):
-            source = draw(sort(source))
+        # Decide how many Filters and Sorts we're going to apply
+        num_filters = draw(st.integers(min_value=1, max_value=6))
```

Comment:
Do we want a sorted frame to also always be filtered at least once?  The previous `many_rows_per_patient_frame()` source could be a simple `select_table()` with no filters as well as a filtered table

Reply:
Oh, good point. I guess just reducing `min_value` to 0 here will do what we want, right?